private-media-framework/draft-ietf-perc-private-media-framework.md

```diff
@@ -959,6 +971,11 @@ Thus what the media source said can be substantially delayed at the
 receiver with the receiver believing that it is what was said just
 now, and only delayed due to transport delay.
 
+While this attack cannot be eliminated entirely, its effectiveness
+can be reduced by re-keying the conference periodically since such
```

Comment:
I suggest shorten/reword this to " ... its effectiveness can be reduced by re-keying the conference periodically, since significantly-delayed media encrypted with expired keys would not be decrypted by endpoints.

Reply:
Thanks! Done.